sdk/go/x/auto/test/testproj/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: testproj
```

Comment:
Nit: give a more specific name?

Reply:
Agreed generally, but going to defer for now as this shared across multiple tests and a generic name fits for that.